src/status_im/ui/screens/hardwallet/setup/styles.cljs

```diff
@@ -106,11 +148,13 @@
 
 (def puk-code-explanation-text
   {:font-size   15
+   :width       "90%"
+   :text-align  :center
    :padding-top 5
    :color       colors/gray})
 
-(def puk-code-numbers-container
-  {:width           369
+(defn puk-code-numbers-container [width]
+  {:width           (* 0.9 width)
```

Comment:
Is that parent's `width`? Can't it be just `"90%"` then?

Reply:
@rasom `width` is a window width here. For some reason "90%" gives more width than `(* 0.9 width)` (check "puk code" on 2nd screenshot, it's getting too close to the edge. 1st screenshot shows `(* 0.9 width)` solution )

<img width="539" alt="screenshot 2018-10-02 13 27 11" src="https://user-images.githubusercontent.com/23836/46344170-eb27d000-c648-11e8-97d7-48fc6c0c6d68.png">
<img width="525" alt="screenshot 2018-10-02 13 28 02" src="https://user-images.githubusercontent.com/23836/46344172-eb27d000-c648-11e8-8d23-91522ffc789d.png">

Reply:
I was trying "90%" value at first, but it didn't work so i switched to dynamic calculation to not spend too much time on researching why it doesn't work.

Reply:
@rasom do you see any issues with calculating width based on window width?

Reply:
Remove dynamic width calculation based on window width.

Reply:
I don't see any issue, just suggesting